tests/console/validate_vnc_avail.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that after remote installation, the VNC connection
+# between client and host, is available without the use of vnc password.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use mmapi;
+use y2lan_restart_common "close_xterm";
+
+# Returns the first leased IP, in /var/lib/dhcp/db/dhcpd.leases file
+sub get_lease_ipaddr {
+    my $lease_ip;
+    my $dhcp_leases_file = script_output("cat /var/lib/dhcp/db/dhcpd.leases\n");
+    for my $line (split /\n/, $dhcp_leases_file) {
+        if ($line =~ /^lease\s+\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/) {
+            (undef, $lease_ip) = split(/\s/, $line);
+            return $lease_ip;
+        }
+    }
+}
+
+sub start_vncviewer {
+    my $args = shift;
+    $args->{port} //= 1;
+    x11_start_program("vncviewer -fullscreen $args->{server_ip}:$args->{port}",
+        target_match => [qw(displaymanager vnc_certificate_warning remote_master_password)]);
```

Comment:
'remote_master_password' needles is not needed here neither in line 42, only in line 45 due to `match_has_tag` is relative to the last `assert_screen` AFIR.

Reply:
Yes, I had something different in my mind, like adding extra checks in case password is asked and there are no certificate warnings, but I would have to set 2 additional elsif(match_has_tags "remote_passwd") die .... and I am not sure if that would serve any purpose. I guess it's more simple to remove it.